### DISCUSSION TITLE
How To Approve Token Transfers Without Spending Gas Using EIP-2612

### DISCUSSION THREAD

**Discussion by DerXanRam** [mhw0/libethc#29] [Q&A]
Hello bro.... i u doing.... I'm trying to swap using 1inch api endpoint and it needs  permit to spend my balance and it(1inch) refers [this article](https://eips.ethereum.org/EIPS/eip-2612) about how to do that. I try to understand it, but very hard for me. Please help

**Comment by mhw0** [comment]
Hello. If it's related to the library, just tell me please what's not working.

**Reply by DerXanRam** [reply]
I don't think so it is related. sorry for that

**Comment by Brandiesuch** [comment]
very entertaining information
